src/utils.js

```diff
@@ -95,6 +98,35 @@ export const courseToRegister = (enrollment, course, student) => ({
     "preLoaded": false,
     "submitPending": false,
 });
+
 export const truncateStrings = (string, length) => string.length > length
     ? `${string.slice(0, length - 3).trim()}...`
     : string;
+
+const isoStringToUTCDate = (dateString) => {
+    let newDate = new Date("2020-01-01");
+    console.log(Number(dateString.substring(5,7)));
+    newDate.setFullYear(Number(dateString.substring(0,4)));
+    newDate.setMonth(Number(dateString.substring(5,7)));
+    newDate.setDate(Number(dateString.substring(8,10)));
+
```

Comment:
None of this code does anything, as it does not perform side effects nor is it used in the return value.